### PR TITLE
ZK-3949: Setting image attribute of caption to empty string or null c…

### PR DESCRIPTION
**Pull request by klyvechen**
…ause it to display a missing image

### CODE REVIEW

File: zktest/src/org/zkoss/zktest/test2/B85_ZK_3949VM.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.test2;
+
+import org.zkoss.bind.BindUtils;
+import org.zkoss.bind.annotation.Command;
+
+public class B85_ZK_3949VM {
+	private String image = "http://res.publicdomainfiles.com/pdf_view/63/13546045816442.png";
+
```
Comment (DevChu) [inline zktest/src/org/zkoss/zktest/test2/B85_ZK_3949VM.java]: it is a risk for the link to outside web resource

File: zul/src/archive/web/js/zul/LabelImageWidget.js [hunk 2]
```diff
@@ -52,13 +52,15 @@ zul.LabelImageWidget = zk.$extends(zul.Widget, {
...
+				jqn = jq(n);
 			if (n) {
 				var img = v || '';
 				if (jq.nodeName(n, 'img')) // ZK-1100
 					n.src = img;
 				else
-					jq(n).css('background-image', 'url(' + img + ')');
+					jqn.css('background-image', 'url(' + img + ')');
+				jqn[(img == '') ? 'hide' : 'show']();
 			} else if (this.desktop) //<IMG> might not be generated (Bug 3007738)
```
Comment (DevChu) [inline zul/src/archive/web/js/zul/LabelImageWidget.js]: img == '' -> !img

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**